app/src/main/java/org/ole/planet/myplanet/service/UploadManager.java

```diff
@@ -46,6 +52,30 @@ public static UploadManager getInstance() {
 
     public UploadManager(Context context) {
         dbService = new DatabaseService(context);
+        this.context = context;
+    }
+
+    public void uploadActivities(SuccessListener listener) {
+        ApiInterface apiInterface = ApiClient.getClient().create(ApiInterface.class);
+        SharedPreferences pref = PreferenceManager.getDefaultSharedPreferences(context);
+        try {
```

Comment:
@yuviii  still not working 

<img width="579" alt="Screen Shot 2019-03-10 at 18 05 08" src="https://user-images.githubusercontent.com/9145217/54084894-2ef0d500-435f-11e9-813a-49f5c40eef84.png">

The problem is you are using default SharedPreference here, but the time is saved in private SharedPreference . Please check constructor of SyncManager and test properly.